client/my-sites/plugins/plugins-list/index.jsx

```diff
@@ -416,9 +425,17 @@ export class PluginsList extends Component {
 			</div>
 		);
 
+		let isJetpackIncluded = false;
+
+		plugins.filter( this.isSelected ).forEach( ( plugin ) => {
+			if ( plugin.slug === 'jetpack' ) {
+				isJetpackIncluded = true;
+			}
+		} );
+
```

Comment:
Nitpicking here.

In JavaScript, when we need to iterate over an array to set a single boolean variable, we prefer other methods than `forEach`.

For instance, arrays have a method called `some`, which returns true or false depending on the result of running a function over the entire array. If the result of the function is true for one value, then the entire expression returns true. Otherwise, it returns false.

In concrete, we can reduce the entire expression into something like this. 
```
const isJetpackIncluded = plugins.filter( this.isSelected ).some( ( { slug } ) => slug === 'jetpack' ) ;
```

I feel that most JavaScript developers will find the latter expression clearer since functional programming is encouraged. Obviously, this might not be true for everyone.